samgeo/samgeo3.py

```
@@ -66,6 +68,28 @@
 SAM31_CONFIG_NAME = "config.json"
 
 
+def _tensor_to_numpy(tensor):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_tensor_to_numpy`

(ANN202)